components/forms/main.html

```diff
@@ -0,0 +1,66 @@
+    <section class="desc flex flex-col">
+      <p class="desc m-md p-sm fs-m">
+        Forms are used to collect users data and pass it for processing or storing. They are very crucial and widely used across the web.
+      </p>      
+    </section>
+    <!-- default -->
+    <section class="prev flex">
+      <h4 class="title fs-l" id="input-std">Preview: General Form</h4>
+      <form class="form card p-sm">
+        <div class="h3  text-center fw-semibold m-xs">LOGIN</div>
+        <div class=" m-xs">
+          <label class="fs-s">
+            Email 
+          </label>
+          <input 
+                 type="email" 
+                 placeholder="tushar@furnhouse.com"
+                 class="input full-width px-sm py-xs my-xs rounded-s"
+           />
+        </div>
+        <div class=" m-xs">
+          <label class="fs-s">
+            Password 
+          </label>
+          <input 
+                 type="password" 
+                 placeholder="********"
```

Comment:
you can use placeholders to give a message like - "Enter your password" just a suggesstion.

Reply:
Thanks for the suggestion @rkapoor10, I appreciate that.
But in the Figma template, it was like this only so I am using this.

Reply:
Done